### PR TITLE
docs: the LTR icon uses svg to better fit the dark theme

### PR DESCRIPTION
**Pull request by KuangPF**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     docs: the LTR icon uses svg to better fit the dark theme'      |
| 🇨🇳 Chinese |     docs: LTR 图标使用 svg，更好适配暗黑主题     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 550f46b</samp>

This pull request adds custom icons for the direction switcher in the `Header` component of the documentation theme. It creates two new files, `LTRIcon.tsx` and `RTLIcon.tsx`, that export SVG-based React components for the icons.

### <samp>🤖 Generated by Copilot at 550f46b</samp>

*  Add new components for left-to-right and right-to-left icons ([link](https://github.com/ant-design/ant-design/pull/42798/files?diff=unified&w=0#diff-329de16fa769871b7fbf32648ff7faaed48bc1361504818965c34e351ec635b2R1-R14), [link](https://github.com/ant-design/ant-design/pull/42798/files?diff=unified&w=0#diff-23f20a2718dbbb07138d8a949d81c8596383b4214e1ab2d8094ae777f5c3509bR1-R14))
*  Import `LTRIcon` and `RTLIcon` into `Header` component ([link](https://github.com/ant-design/ant-design/pull/42798/files?diff=unified&w=0#diff-2b1224dbeb7d20f1cb1b4590eb12b68104c605d2a67f0b0bd4fd9d8b0100a728L10-R20))
*  Apply `dataDirectionIcon` CSS class to icons to set width ([link](https://github.com/ant-design/ant-design/pull/42798/files?diff=unified&w=0#diff-2b1224dbeb7d20f1cb1b4590eb12b68104c605d2a67f0b0bd4fd9d8b0100a728R100-R102))
*  Replace `img` elements with `LTRIcon` and `RTLIcon` in `SwitchBtn` component ([link](https://github.com/ant-design/ant-design/pull/42798/files?diff=unified&w=0#diff-2b1224dbeb7d20f1cb1b4590eb12b68104c605d2a67f0b0bd4fd9d8b0100a728L314-R321))

### CODE REVIEW

**Comment by KuangPF** [comment]
RTL-dark
LTR-Dark

File: .dumi/theme/common/RTLIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Icon from '@ant-design/icons';
+import React from 'react';
+
+const SVGIcon: React.FC = () => (
+  <svg viewBox="0 0 1024 1024" fill="currentColor">
+    <path d="M256 64l512 0 0 128-128 0 0 768-128 0 0-768-128 0 0 768-128 0 0-448c-123.712 0-224-100.288-224-224s100.288-224 224-224zM960 896l-256-224 256-224z" />
```
Comment (li-jia-nan) [inline .dumi/theme/common/RTLIcon.tsx:6]: 建议把 `direction` 从外面传进来，这样就不需要封装两遍了：

```tsx
<svg viewBox="0 0 1024 1024" fill="currentColor">
  <path d={direction === 'rtl' ? '…' : '…'} />
</svg>
```

Comment (KuangPF) [reply .dumi/theme/common/RTLIcon.tsx]: 已处理。

Comment (li-jia-nan) [reply .dumi/theme/common/RTLIcon.tsx]: OK

**Review by li-jia-nan — APPROVED** [review]

File: .dumi/theme/common/RTLIcon.tsx (hunk 1 above)
Comment (KuangPF) [reply .dumi/theme/common/RTLIcon.tsx]: 秒回复👍

File: .dumi/theme/common/DirectionIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import type { DirectionType } from 'antd/es/config-provider';
+import React from 'react';
+
+const ltrD =
+  'M448 64l512 0 0 128-128 0 0 768-128 0 0-768-128 0 0 768-128 0 0-448c-123.712 0-224-100.288-224-224s100.288-224 224-224zM64 448l256 224-256 224z';
+const rtlD =
+  'M256 64l512 0 0 128-128 0 0 768-128 0 0-768-128 0 0 768-128 0 0-448c-123.712 0-224-100.288-224-224s100.288-224 224-224zM960 896l-256-224 256-224z';
+
+const DirectionIcon: React.FC<{ direction: DirectionType }> = (props) => (
+  <Icon {...props}>
```
Comment (li-jia-nan) [inline .dumi/theme/common/DirectionIcon.tsx:11]:
```suggestion
  <Icon {...omit(props, ['direction'])}>
```
这里不应该全部透传进来吧

**Pull request merged**